nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -506,6 +506,8 @@
   [Prisma ORM upgrade guide](https://www.prisma.io/docs/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-6)
   for more information.
 
+- `ociTools.buildContainer` has been removed in favor of `ociTools.buildImage`.
```

Comment:
```suggestion
- []{#ociTools-removed} `ociTools.buildContainer` has been removed in favor of `ociTools.buildImage`.
```

This needs to live in the Nixpkgs release notes for the redirect to work, and this makes sense because it's a Nixpkgs library.

Reply:
Should we also add an entry regarding the general new OCI interface? If so, should there be a new section created for it? 
Also, the "library" deprecations lists non-lib stuff as of now (e.g. Plasma 5). I moved these to a general deprecations section for now.

Reply:
Oh yes sure, this is an important change that we should make users aware of!

Reply:
Added a "highlights" section, similar to what we have in the NixOS release notes.